R/zzz.R

```diff
@@ -116,6 +116,17 @@ move_env_elements(RPolarsExpr, pl, c("lit"), remove = FALSE)
 
 
 .onLoad = function(libname, pkgname) {
+  # Auto limit the max number of threads used by polars
+  if (
+    isFALSE(cargo_rpolars_feature_info()[["disable_auto_limit_max_threads"]]) &&
+      !isFALSE(getOption("polars.limit_max_threads")) &&
```

Comment:
```suggestion
      isTRUE(getOption("polars.limit_max_threads")) &&
```

Reply:
This option is intended to only work when explicitly specified as `FALSE`